microsoft/nni
# [webui] Improve trial parameter for retiarii experiment

### Description ###
for retiarii experiment:
- show trial normal parameter
  
![image](https://user-images.githubusercontent.com/61399850/142824684-03d6817d-b5e5-40a5-9a73-723796897953.png)


- show real trial parameter by clicking `Original parameters` such as this picture:

![image](https://user-images.githubusercontent.com/61399850/142824746-9b1cf1f4-2ffc-40cb-89f6-e9acb076d465.png)



### Checklist ###
  - [x] these PR should be merged after the PR: https://github.com/microsoft/nni/pull/4304

### How to test ###

1. run example `examples/nas/multi-trial/mnist`, 
- [ ] see the hyper-parameter graph
- [x] see trial parameter in the table (oview && detail table)
- [x] copy trial parameters
- [x] confirm trail `Compare` model in the detail page
- [ ] confirm `Default metric | Intermediate result` graph's tooltip

2. [x] run nested example and see the website
3. [x] run mnist example and see the website
